webapp/portlet/src/main/webapp/vue-apps/attach-image/components/AttachImageComponent.vue

```diff
@@ -0,0 +1,62 @@
+<!--
+ This file is part of the Meeds project (https://meeds.io/).
+ 
+ Copyright (C) 2020 - 2023 Meeds Association contact@meeds.io
+ 
+ This program is free software; you can redistribute it and/or
+ modify it under the terms of the GNU Lesser General Public
+ License as published by the Free Software Foundation; either
+ version 3 of the License, or (at your option) any later version.
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ Lesser General Public License for more details.
+ 
+ You should have received a copy of the GNU Lesser General Public License
+ along with this program; if not, write to the Free Software Foundation,
+ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+-->
+<template>
+  <div class="px-3">
+    <file-multi-upload-input @update-images="upadateImage($event)" />
+    <image-items :images="attachedImages" />
+  </div>
+</template>
+<script>
+export default {
+  data: () => ({
+    images: []
+  }),
+  computed: {
+    attachedImages() {
+      return this.images;
+    }
+  },
+  created() {
+    extensionRegistry.registerExtension('activity', 'saveAction', {
+      key: 'attachment',
+      postSave: activity => {
+        if (activity?.id) {
+          const uploadIds = this.images.map(file => file.uploadId).filter(uploadId => !!uploadId);
+          if (uploadIds?.length) {
+            const attachment = {
+              'objectId': activity?.id,
+              'spaceId': activity?.activityStream?.space?.id,
+              'uploadIds': uploadIds,
+              'objectType': 'activity',
+              
+            }; 
+            return this.$fileAttachmentService.createAttachments(attachment)
```

Comment:
Please use attachment report to display more information for user by specifying file name in message and automatically delete files not being uploaded from previewed files, or display the files with error message inside image card.

```suggestion
            return this.$fileAttachmentService.createAttachments(attachment)
            .then(report => ...)
```

Reply:
done!